mpas_analysis/config.default

```diff
@@ -316,13 +317,6 @@ remappedClimSubdirectory = clim/obs/remapped
 # directory where ocean reference simulation results are stored
 baseDirectory = /dir/to/seaice/reference
 
-[icebergObservations]
-## options related to iceberg observations with which the results will be
-## compared
-
-# directory where sea ice observations are stored
-baseDirectory = /dir/to/iceberg/observations
-concentrationAltibergSH = Altiberg/Altiberg_1991-2017.nc
```

Comment:
I think all of this got deleted in some kind of a merge fix.  It needs to be there.